docs/migration/4_0.md

```diff
@@ -1207,6 +1208,15 @@ related labels outside of checkout and the checkout lib is not used, you will ne
 `RoutingService.go` - Removed 2nd argument `query`. Use `extras.queryParams` instead.
 `RoutingService.navigate` - Removed 2nd argument `query`. Use `extras.queryParams` instead.
 
+### RoutingActions ngrx
+The following ngrx actions has been deprecated:
```

Comment:
Shouldn't be `have been deprecated`, as `actions` is a plural form? But I'm not an expert ;)

Reply:
Thanks for noticing. Btw. They are not only deprecated. I decided to remove them (in later commit).
Thanks again and I'll update the docs and schematics comment.